### PR TITLE
Onboard judging controller spec

### PR DESCRIPTION
**Pull request by AkivaGreen**
Added onboard judging update spec and maneuver finding spec
Addresses #15

### CODE REVIEW

File: spec/features/maneuver_finding_spec.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  let!(:record_3) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  context 'finds the previous' do
+    it 'at first participant gets first participant' do
+      when_current_user_is :admin
+      visit maneuver_participant_path(record_1.id)
+      click_link 'Previous participant'
+      expects = 'This is the first participant who completed this maneuver.'
```
Comment (dfaulken) [inline spec/features/maneuver_finding_spec.rb]: `expects` is a kind of odd variable name here - previous PRs have gone for `expected`.

File: spec/features/maneuver_finding_spec.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  let!(:record_2) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  let!(:record_3) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  context 'finds the previous' do
+    it 'at first participant gets first participant' do
```
Comment (dfaulken) [inline spec/features/maneuver_finding_spec.rb]: This isn't a sentence, unfortunately - `it` descriptions really should always start with verbs (that aren't `should`)

File: spec/features/onboard_judging_update_spec.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+describe 'updating onboard judging' do
+  context 'with judge priviledge' do
+    let!(:onboard_judging) { create :onboard_judging, :perfect }
+    it 'updates onboard judging' do
+      when_current_user_is :judge
+      visit select_participant_onboard_judgings_url
+      click_link 'Review'
+      fill_in 'onboard_judging_minutes_elapsed', with: '10'
```
Comment (dfaulken) [inline spec/features/onboard_judging_update_spec.rb]: Is this line necessary?

File: spec/models/maneuver_participant_spec.rb [hunk 4]
```diff
@@ -6,47 +6,47 @@
...
-    it 'should go down by 50 when not completed as designed' do
+    it 'lowers by 50 when not completed as designed' do
       expect { record.update completed_as_designed: false }
         .to change { record.score }
         .by(-50)
     end
 
-    it 'should go down by the sum of each point value multiplied by count' do
+    it 'lowers by the sum of each point value multiplied by count' do
       expect { record.update obstacles_hit: { 1 => 1 } }
```
Comment (dfaulken) [inline spec/models/maneuver_participant_spec.rb]: So this test doesn't make it clear how the `1 => 1` data structure translates into a point deduction. It'd be nice to construct it incrementally with descriptive variable names, just to break it down a bit (same for the following tests).

File: spec/features/onboard_judging_update_spec.rb (hunk 3 above)
Comment (dfaulken) [inline spec/features/onboard_judging_update_spec.rb]: Update: yes it is

**Comment by dfaulken** [comment]
Codeclimate failures are okay.

File: spec/features/maneuver_finding_spec.rb [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+  let!(:record_2) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  let!(:record_3) do
+    create :maneuver_participant,
+           :perfect_score, maneuver: maneuver
+  end
+  context 'finds the previous' do
+    it 'called on first participant gets first participant' do
```
Comment (dfaulken) [inline spec/features/maneuver_finding_spec.rb]: "It called on first participant gets first participant" still isn't a sentence, unfortunately.

Comment (AkivaGreen) [inline spec/features/maneuver_finding_spec.rb]: if we change it to " it 'gets first participant when called on first participant'" or "it 'gets first participant at first participant'" incase the first is too long. Will that syntax be okay?

Comment (dfaulken) [inline spec/features/maneuver_finding_spec.rb]: It sounds like the subordinate clause of that sentence could be a context block.

**Pull request merged**